lib/azure/deploy.rb

```diff
@@ -81,9 +81,11 @@ def create(params)
       if params[:cert_path]
         cert_data = File.read (params[:cert_path])
         @connection.certificates.add cert_data, params[:cert_password], 'pfx', params[:azure_dns_name]
-      elsif(params[:winrm_transport] == "ssl")
-        thumbprint = @connection.certificates.create_ssl_certificate params[:azure_dns_name]
-        params[:ssl_cert_fingerprint] = thumbprint.to_s.upcase
+
+##########  Uncomment the code below to activate automatic certificate generation for ssl communication
```

Comment:
I think we should just leave a TODO: here rather than the old code (we can always grab it from source control later). The comment should just be something like `TODO: generate certificates for ssl listener`